utils/constants/Data.ts

```diff
@@ -0,0 +1 @@
+export const BASE_DATA_URI = 'https://dimo.zone/vehicle/data/';
```

Comment:
One interesting thing is that our URLs right now look like `/vehicle/6/status`.

My guess is we'll just have to adjust.

Reply:
Does not hurt change it in the code. What option you think is the easiest?